atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraTimestampBoundStore.java

```diff
@@ -140,13 +145,16 @@ private void cas(Client client, Long oldVal, long newVal) {
         }
         if (!result.isSuccess()) {
             String msg = "Timestamp limit changed underneath us (limit in memory: " + currentLimit
+                    + ", stored in DB: " + getCurrentValues(result)
```

Comment:
I think we should also log the expected `oldVal` & `newVal` as part of this message for consistency, something along the lines of `"unable to CAS from " + oldVal + " to " + newVal`